common/src/main/java/com/microsoft/identity/common/internal/commands/BaseNativeAuthCommand.kt

```diff
@@ -64,8 +64,7 @@ abstract class BaseNativeAuthCommand<T>(
         return false
     }
 
-    // TODO update this once telemetry is done
```

Comment:
Are you integrating any telemetry in this PR?

Reply:
yes. we had made some changes for telemetry after our last merge and this PR includes those changes.